README.md

```diff
@@ -1 +1,15 @@
 # Gra w statki
```

Comment:
Możesz dopisać, że na MacOS jest automatycznie zainstalowany, więc nie trzeba nic instalować

Reply:
A wiesz coś dokładniej? Jakie wersje MacOS czy coś? Sprawdzałeś jakoś na czystym środowisku?

Reply:
Xcode w wersji powyzej 12.3 ma automatycznie wgrane, a Xcode jest potrzebny do uzywania czegokolwiek zwiazanego z programowaniem na macu, wiec jak ktos nie ma xcode to zapewne nie ma kompilatora c++ wgranego rowniez. 
Jedyne co sie kloci to #include <bits/stdc++.h>, ktorego na macu nie ma, wiec w sumie mozesz nic nie dopisywac o macu bo i tak sie nie skompiluje

Reply:
Wersja Xcode 12.3 jest na kazdym macu na wersji 12.3 i w gore, obecnie jest 13.1 najnowsza

Reply:
Dobra, to ja bym kwestię obsługi MacOS zostawił na końcówkę projektu, ok? Mogłaby być jakaś sekcja w readme. No chyba, że ty chcesz dodać to do readme, to spoko